tests/auth_test.rb

```diff
@@ -74,4 +79,34 @@ def test_invalid_authenticate_request_header
       PassageHeaderClient.auth.authenticate_request(no_header_request)
     end
   end
+
+  def test_create_magic_link
+    magic_link =
+      PassageClient.auth.create_magic_link(
+        email: 'chris@passage.id',
```

Comment:
these can 100% not be my name lol. i think this is technically referring to the first chris, but not necessary to use a person's name 😅

Reply:
😆 I guessssssssss